### ISSUE TITLE
Signal.shape deprecated

### ISSUE THREAD

**Issue by jeanthom** [lambdaconcept/lambdaUSB#3]
It affects lambdaUSB...

```
Traceback (most recent call last):
  File "fomu-audio.py", line 81, in <module>
    platform.build(USBBlinker())
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/build/plat.py", line 78, in build
    plan = self.prepare(elaboratable, name, **kwargs)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/build/plat.py", line 118, in prepare
    fragment = Fragment.get(elaboratable, self)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/hdl/ir.py", line 39, in get
    obj = obj.elaborate(platform)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/hdl/dsl.py", line 520, in elaborate
    fragment.add_subfragment(Fragment.get(self._named_submodules[name], platform), name)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/hdl/ir.py", line 39, in get
    obj = obj.elaborate(platform)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/hdl/dsl.py", line 520, in elaborate
    fragment.add_subfragment(Fragment.get(self._named_submodules[name], platform), name)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/nmigen/hdl/ir.py", line 39, in get
    obj = obj.elaborate(platform)
  File "/home/jeanthomas/.local/lib/python3.7/site-packages/lambdausb/ctl.py", line 149, in elaborate
    rx_skip   = Signal.range(3, reset=2)
AttributeError: type object 'Signal' has no attribute 'range'
```

**Comment by povauboin** [comment]
I merged 63fc8499 into master.

**Comment by jeanthom** [comment]
Thanks!
